content/CORETutorials/JailsPluginsVMs/Plugins/MinIOPlugin.md

```diff
@@ -117,30 +137,72 @@ Due to incompatibility between versions, `mc` is unable to export configuration
         The Minio plugin must use different ports than the S3 service, which uses 9000 and 9001 by default.
         Before selecting port numbers, refer to [Default Ports](https://www.truenas.com/docs/references/defaultports/).
-    6. Record the ports used for the Minio plugin.
-    7. Click **Save**
+        Record the ports used for the Minio plugin.
+
+    f. Click **Save**
+
 2. Ensure that the destination dataset created in [First Steps](#first-steps) is mounted on the Minio Plugin, and not the dataset used for the S3 service.
-   1. Go to **Plugins** and locate the installed Minio plugin or go to **Jails** and locate the jail matching the **Jail Name** entered during [installation of the Minio plugin](#installing-the-minio-plugin).
-   2. Click <i class="material-icons" aria-hidden="true" title="Expand">chevron_right</i> to expand Minio details and management options.
-   3. Click <span class="material-icons">device_hub</span>&nbsp;**MOUNT POINTS** and ensure configured mount point matches the destination dataset you created in [First Steps](#first-steps).
+
+    a. Go to **Plugins** and locate the installed Minio plugin or go to **Jails** and locate the jail matching the **Jail Name** entered during [installation of the Minio plugin](#installing-the-minio-plugin).
+
+    b. Click <i class="material-icons" aria-hidden="true" title="Expand">chevron_right</i> to expand Minio details and management options.
+
+    c. Click <span class="material-icons">device_hub</span>&nbsp;**MOUNT POINTS** and ensure configured mount point matches the destination dataset you created in [First Steps](#first-steps).
         If needed, follow the instructions in [Setting Up Jail Storage]({{< relref "SettingUpJailStorage.md" >}}) to mount the destination dataset.
+
 3. Use the **MinIO Console** to recreate the configuration settings from the S3 service MinIO deployment in the Minio plugin deployment.
-   1. Go to the MinIO web UI portal for the S3 service deployment and note existing configuration settings including users, groups, access keys, and all other MinIO settings.
-   2. Go to **Plugins** and locate the installed Minio plugin or go to **Jails** and locate the jail matching the **Jail Name** entered during [installation of the Minio plugin](#installing-the-minio-plugin).
-   3. Click <i class="material-icons" aria-hidden="true" title="Expand">chevron_right</i> to expand Minio details and management options.
-   4. Click <span class="material-icons">play_arrow</span>&nbsp;**START** to start the plugin (if needed) and then click <span class="material-icons">settings</span>&nbsp;**MANAGE** to go to the **MinIO Console** and log in.
-   5. Configure MinIO settings in the new deployment to match those of the existing deployment.
+
+    a. Go to the MinIO web UI portal for the S3 service deployment and note existing configuration settings including users, groups, access keys, and all other MinIO settings.
```

Comment:
One thing that may be worth mentioning is that the secret key portion of an access key cannot be viewed after initial creation. 

If a customer had been using the S3 service for sometime, they may have a number of clients already configured to use a specific access key/secret key combo in conjunction with the S3 service as a storage target . If they did not record/save their access key/secret key, there is a possibility they will also need to reconfigure any clients using MinIO as a target with new access keys/secret keys. If they saved all of their key pairs they should be able to input them, but in worse case scenario they may have to configure a number of other devices/services that were using the S3 service as a storage target with new keypairs before it can properly communicate with the new MinIO plugin deployment.

Potentially not worth listing that out in its entirety, but it is definitely something support should be aware of, otherwise they may assist with migration to current MinIO and end up severing communication to MinIO until clients are configured with new key pairs.